gui/common_qt_functions.py

```diff
@@ -69,21 +78,30 @@ def setAzureSettings(self):
             'auth-resource': '',
             'update-md5': False,
             'validate-md5': False,
-            'virtual-directory': False,
```

Comment:
I thought this was causing issues and we set it to False for a reason, did we fix this so it can be true?

Reply:
I think this setting was just flagged false because most booleans are false by default. This one is true by default. And I think setting it true actually makes the code more compatible (it allows blobs to show up whose parent folders don't have a folder marker in cloud storage).

Reply:
Agreed.